docs/articles/expensify-classic/workspace-and-domain-settings/Categories.md

```diff
@@ -1,5 +1,149 @@
+
+To configure category approvers:
+
+1. Go to Settings > Workspace > Group > _[Select Workspace]_ > Categories
+2. Click Edit Rules next to the category name that requires a category approver and use the "Approver" field
+   
+The expense will route to this Category approver, before following the approval workflow set up in the People table
+
+## Auto-categorize card expenses with default categories
+
+If you're importing card transactions, **Default Categorization** will provide a massive benefit to your company's workflow by **automatically coding** expenses to the proper GL.
+- Once configured according to your GL, the default category will detect the type of merchant for an expense based on its Merchant Category Code (MCC) and associate that expense with the proper GL account. 
+- This time-saver keeps employees from having to manually code expenses and provides admins with the peace of mind that the expenses coming in for approval are more reliably associated with the correct GL account. 
+- Best of all, this works for personal and company cards alike!
+  
+## Setting up Default Categories
+
+1. First, go to **Settings** and select the **group Workspace** that you want to configure.
+2. Go to the **Categories** tab and scroll down to **Default Categories**.
+3. Under the **Category** column, select the account that is most closely associated with the merchant group for all groups that apply to expenses that you and your coworkers submit. If you are unsure, just leave the group **Uncategorized** and the expense will not come in pre-categorized.
+4. You're well on your way to expense automation freedom!
+   
+## Default Categories based on specific MCC codes
+
+If you require more granular detail, the MCC Editor gives you even greater control over which MCC Codes are assigned to which Categories. The MCC Editor can be found just below the Default Categories table.
+
+## Implicit categorization
+Over time, Expensify will learn how you categorize certain merchants and then automatically apply that category to the same merchant in the future.
+
+- You can always change the category; we'll try to remember that correction for next time!
+- Any Expense Rules you have set up will always take precedence over implicit categories. 
+- Implicit categorization will **only** apply to expenses if you have **not** explicitly set a category already.  Changing the category on one expense does not change it for any other expense that has an explicit category already assigned.
+  
+This built-in feature will only use the categories from the currently active workspace on your account. You can change the active workspace by clicking your account icon in the app and selecting the correct workspace name before you SmartScan.
+  
+## Category violations
+Category violations can happen for the following reasons:
+
+- An employee categorized an expense with a category not included in the workspace's categories. This would throw a "category out of workspace" violation
```

Comment:
```suggestion
- An employee categorized an expense with a category not included in the workspace's categories. This would throw a "category out of workspace" violation.
```